src/Admin/ApproachAdmin.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace AppBundle\Admin;
```

Comment:
What about moving these admins in a specific namespace ? 
e.g. : `AppBundle\Admin\ProgrammaticFoundation`

like here : https://github.com/EnMarche/en-marche.fr/tree/master/src/Admin/ApplicationRequest

Reply:
Great idea thanks, done.